agent/backend/pktvisor/pktvisor.go

```diff
@@ -62,7 +66,7 @@ func (p *pktvisorBackend) GetStartTime() time.Time {
 
 func (p *pktvisorBackend) SetCommsClient(agentID string, client mqtt.Client, baseTopic string) {
 	p.mqttClient = client
-	p.metricsTopic = fmt.Sprintf("%s/m/%c", baseTopic, agentID[0])
+	p.metricsTopic = fmt.Sprintf("%s/m/%s", baseTopic, agentID)
```

Comment:
why the switch to full agentID?

Reply:
hm, this was leading to an error, and the message was not sending, not sure how it was before, is this just the first part of the agentID?